### PR TITLE
feat: [PIE-5071]: add exec inputs support to all approvals

### PR DESCRIPTION
**Pull request by vkbansal-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 33 Issues 
* Code issue 33 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10843)

**Comment by vkbansal-harness** [comment]
retrigger titlecheck

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/70-pipeline/components/execution/StepDetails/views/ServiceNowCreateUpdateView/ServiceNowCreateUpdateView.tsx [hunk 1]
```diff
@@ -38,16 +52,47 @@ export function ServiceNowCreateUpdateView(props: ServiceNowCreateUpdateViewProp
...
+      }}
+    >
+      {shouldShowExecutionInputs ? (
+        <Tabs.Tab
+          id={ApprovalStepTab.STEP_EXECUTION_INPUTS}
+          title={getString('pipeline.runtimeInputs')}
+          panel={<ExecutionInputs step={step} />}
+        />
+      ) : null}
+      <Tabs.Tab
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/execution/StepDetails/views/ServiceNowCreateUpdateView/ServiceNowCreateUpdateView.tsx:84]: Is `<Tabs.Tab` giving you correct styles of tabs?

Comment (vkbansal-harness) [reply src/modules/70-pipeline/components/execution/StepDetails/views/ServiceNowCreateUpdateView/ServiceNowCreateUpdateView.tsx:84]: I just used whatever was already being used.

Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/execution/StepDetails/views/ServiceNowCreateUpdateView/ServiceNowCreateUpdateView.tsx:84]: AFAIK, We should use Tabs from uicore
import { Tab, Tabs } from '@harness/uicore'

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good
1 comment https://github.com/harness/harness-core-ui/pull/10843/files#r956970839

**Pull request merged**